src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -437,6 +428,161 @@ func (repo *MarketplaceCmdRepo) InstallItem(
 	)
 }
 
+func (repo *MarketplaceCmdRepo) moveSelectedFiles(
+	sourceDir valueObject.UnixFilePath,
+	targetDir valueObject.UnixFilePath,
+	fileNames []valueObject.UnixFileName,
+	shouldKeepInstead bool,
+) error {
+	params := ""
+	if len(fileNames) > 0 {
+		fileNameFilterParams := "-name \"" + fileNames[0].String() + "\""
+		for _, fileToIgnore := range fileNames[1:] {
+			fileNameFilterParams += " -o -name \"" + fileToIgnore.String() + "\""
+		}
+		params = "\\( " + fileNameFilterParams + " \\)"
+	}
+
+	flags := []string{"-maxdepth 1"}
+	if shouldKeepInstead {
+		flags = append(flags, "-mindepth 1", "-not")
```

Comment:
When the operation does not include the `-not` flag, it ignores the root directory of the search. For example, using `find` without `-not` in the `/app/html` directory will not list `/app/html`. However, if I add the `-not` flag without specifying `-mindepth 1`, it will list `/app/html`.

Anyway, adding `-mindepth 1` to the original command, regardless of the `-not` flag, will not affect the result. So, I think it's better to keep this flag as the command's default.